src/components/grid/grid.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import customProperties from '../../../materials/custom-properties.json';
+import filterDataAttributes from '../../utils/filter-data-attributes';
+
+// TODO: check with design if those "breakpoints" are ok
+const minColumnSizes = {
+  1: '600px',
+  2: '450px',
+  3: '350px',
+  4: '275px',
```

Comment:
Our possible values are all over the place right now. We sometimes use numbers, words or shirt sizes.

I'd use shirt-sizes for now, so `s`, `m`, `l`, `xl`

Reply:
Yeah, that's one of the things we need to check with the design team.